cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -1060,17 +1239,21 @@ class two_table_comparator {
    * Similarly, `F(rhs_index_type i, lhs_index_type j)` returns true if and only if row `i` of the
    * right table compares equal to row `j` of the left table.
    *
-   * @tparam Nullate A cudf::nullate type describing whether to check for nulls
-   * @param nullate Indicates if either input column contains nulls
-   * @param nulls_are_equal Indicates if nulls are equal
+   * @tparam Nullate A cudf::nullate type describing whether to check for nulls.
+   * @tparam PhysicalElementComparator A equality comparator functor that compares individual values
+   * rather than logical elements, defaults to a `NaN == NaN` equality comparator.
+   * @param nullate Indicates if either input column contains nulls.
```

Comment:
"any input column" or "either input table"? I like "any input column" because it's true for both the self and two-table comparators.
```suggestion
   * @param nullate Indicates if any input column contains nulls.
```

Reply:
Fixed across the board, should be `any column` except inside the element comparator which only concerns itself with 2 columns.